lib/jekyll-ical-tag/calendar_parser.rb

```diff
@@ -12,14 +11,36 @@ def initialize(raw_feed)
       end
 
       def events
-        @events ||= parsed_feed.sort { |event1, event2| event1.dtstart <=> event2.dtstart }
-                               .map { |event| Jekyll::IcalTag::Event.new(event) }
+        @events ||= parsed_feed.sort_by(&:dtstart)
+          .map { |event| Jekyll::IcalTag::Event.new(event) }
       end
 
       private
 
+<<<<<<< Updated upstream
       def parsed_feed
         Icalendar::Event.parse(@raw_feed)
+=======
+      def parsed_events
+        events = Icalendar::Event.parse(@raw_feed)
+
+        recurring_events =
+          events.flat_map do |event|
+            next unless event.rrule.present?
+
+            event
+              .occurrences_between(@recurring_start_date, @recurring_end_date).drop(1)
```

Comment:
Only actual change I made is by adding `.drop(1)` here the rest are from the main branch being merged to the feature